src/InputWithLabel.js

```diff
@@ -0,0 +1,33 @@
+import React, { useRef, useEffect } from 'react';
+
+const InputWithLabel = ({ name, type, id, placeholder, value, onChange, required, children }) => {
+    const inputRef = useRef(null);
+
+    useEffect(() => {
+        if (inputRef.current) {
+            inputRef.current.focus();
+        }
+    });
+
+    return (
+        <div>
+            <label htmlFor={id}>{children}</label>
+            <input
+                ref={inputRef}
+                name={name}
+                type={type}
+                id={id}
+                placeholder={placeholder}
+                value={value}
+                onChange={onChange}
+                required={required}
+            />
+        </div>
```

Comment:
Again no need to change anything, this is just more what the directions were asking for. I will mention that this would have been a good place to use a `Fragment` instead of a `<div>`

```
    <>
      <label htmlFor="inputField">{props.children}</label>
      <input
        type="text"
        id="inputField"
        name="title"
        value={props.todoTitle}
        onChange={props.handleTitleChange}
        required
        ref={inputRef}
      />
    </>

```